src/wallet/sqlite.cpp

```diff
@@ -5,12 +5,20 @@
 #include <wallet/sqlite.h>
 
 #include <logging.h>
+#include <sync.h>
 #include <util/strencodings.h>
 #include <util/translation.h>
 #include <wallet/db.h>
 
 #include <sqlite3.h>
 #include <stdint.h>
+#include <unordered_set>
+
+namespace {
+    RecursiveMutex cs_sqlite;
```

Comment:
In commit "Introduce g_file_paths" (d038d04a4e3e0522a51cf0d39110749f8395f61a)

Minor: Pretty sure we can get rid of these globals with more sane loading code in the wallet, but in any case could consider switching RecursiveMutex to Mutex if possible and switching `cs_sqlite` to `g_sqlite_mutex` to follow newer conventions

Reply:
Renamed